routes/channel.js

```diff
@@ -19,6 +19,9 @@ router.get('/:id/tree', channelLoad, getStatus.bind(null, true))
 router.get('/:id/validator-messages', getValidatorMessages)
 router.get('/:id/validator-messages/:uid/:type?', channelIfExists, channelLoad, getValidatorMessages)
 
+// event information
+router.get('/:id/events', authRequired, channelIfExists, channelLoad, getEvents)
```

Comment:
we should rename this to event-aggregates, cause we're getting event aggregates

Reply:
okay will change it